yarn.lock

```diff
@@ -6068,6 +6083,22 @@ cli-truncate@^0.2.1:
     slice-ansi "0.0.4"
     string-width "^1.0.1"
 
+cli-truncate@^2.1.0:
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/cli-truncate/-/cli-truncate-2.1.0.tgz#c39e28bf05edcde5be3b98992a22deed5a2b93c7"
+  integrity sha512-n8fOixwDD6b/ObinzTrp1ZKFzbgvKZvuz/TvejnLn1aQfC6r52XEx85FmuC+3HI+JM7coBRXUvNqEU2PHVrHpg==
+  dependencies:
+    slice-ansi "^3.0.0"
+    string-width "^4.2.0"
+
+cli-truncate@^3.1.0:
```

Comment:
I'm not doing JS dev here so excuse the intrusion, but how/why can there be two versions of a thing?

Reply:
Packages usually depend on other packages and they usually require a specific version. Sometimes you'll have 2 packages that depend on the same package, but different versions. The lockfile contains which version(s) to use to satisfy all packages. You can use `yarn why <package>` to see why a certain package versions are needed.

Here's what we get with `cli-truncate`:
```
$ yarn why cli-truncate
yarn why v1.22.18
[1/4] Why do we have the module "cli-truncate"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "cli-truncate@0.2.1"
info Has been hoisted to "cli-truncate"
info Reasons this module exists
   - Hoisted from "cypress#listr#listr-update-renderer#cli-truncate"
   - Hoisted from "@vue#cli-plugin-e2e-cypress#cypress#listr#cli-truncate"
   - Hoisted from "@vue#cli-plugin-e2e-cypress#cypress#listr#listr-update-renderer#cli-truncate"
info Disk size without dependencies: "80KB"
info Disk size with unique dependencies: "168KB"
info Disk size with transitive dependencies: "316KB"
info Number of shared dependencies: 6
=> Found "lint-staged#cli-truncate@3.1.0"
info This module exists because "lint-staged" depends on it.
info Disk size without dependencies: "20KB"
info Disk size with unique dependencies: "108KB"
info Disk size with transitive dependencies: "256KB"
info Number of shared dependencies: 6
=> Found "listr2#cli-truncate@2.1.0"
info This module exists because "lint-staged#listr2" depends on it.
info Disk size without dependencies: "20KB"
info Disk size with unique dependencies: "108KB"
info Disk size with transitive dependencies: "256KB"
info Number of shared dependencies: 6
Done in 0.76s.
```